recipes/oneworld/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "oneworld" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 52660254dd1941cebca3f3834df27064fc0f583da87c2358f45ce0ac1d4b46d9
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [py<37]
+
+requirements:
+  host:
+    - python
+    - pip
+    - numpy
```

Comment:
I don't think you need `numpy` to build the package, so you can remove it from `host`